src/powersoftau_export_json.js

```diff
@@ -44,43 +45,40 @@ export default async function exportJson(pTauFilename, verbose) {
 
     await fd.close();
 
-    return stringifyBigIntsWithField(curve.Fr, pTau);
-
-
+    return ffUtils.stringifyBigInts(pTau);
```

Comment:
Why did you change the stringifyBigIntsWithField here?

Reply:
Main reason so that its consistent with the stringify function that `zkey export json` subcommand uses:  https://github.com/iden3/snarkjs/blob/e70413b162ba317cfe5b54c5ab271a72d1bbac5d/src/zkey_export_json.js#L10

Reply:
I'm pretty sure that the ffjavascript `stringifyBigInts` function is just straight broken with new versions of the library because the curves are being calculated as Uint8Arrays but then they don't serialize correctly. I tried getting a straight answer from Jordi but he seems confused by the question, Can we continue to use `stringifyBigIntsWithField`, as I'm 85% sure that's what we always want for the stringify function.